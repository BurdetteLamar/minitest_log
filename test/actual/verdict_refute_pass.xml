<log>
  <summary verdicts='2' failures='0' errors='0'/>
  <section name='Test' method='verdict_refute?' type='pass'>
    <section name='Args=[false]'>
      <verdict method='verdict_refute?' outcome='passed' id='0' message='Message 0'>
        <actual class='FalseClass' value='false'/>
      </verdict>
    </section>
    <section name='Args=[nil]'>
      <verdict method='verdict_refute?' outcome='passed' id='1' message='Message 1'>
        <actual class='NilClass' value='nil'/>
      </verdict>
    </section>
  </section>
</log>
