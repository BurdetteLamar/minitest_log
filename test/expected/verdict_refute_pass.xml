<log>
  <summary_ verdicts='2' failures='0' errors='0'/>
  <section_ name='Test' method='verdict_refute?' type='pass'>
    <section_ name='Args=[false, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute?' outcome='passed' id='0' message='Message 0'>
        <actual_ class='FalseClass' value='false'/>
      </verdict_>
    </section_>
    <section_ name='Args=[nil, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute?' outcome='passed' id='1' message='Message 1'>
        <actual_ class='NilClass' value='nil'/>
      </verdict_>
    </section_>
  </section_>
</log>
